pkg/util/system/system_utils.go

```diff
@@ -18,14 +18,25 @@ package system
 
 import (
 	"strings"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
ref: https://github.com/kubernetes/kubernetes/pull/39112.

Looking at the comments, it is not going to stay in core soon, 😞